client/web/compose/src/components/PageBlocks/RecordBase.vue

```diff
@@ -22,7 +27,8 @@
           :label-cols-xl="options.horizontalFieldLayoutEnabled && '5'"
           :content-cols-md="options.horizontalFieldLayoutEnabled && '6'"
           :content-cols-xl="options.horizontalFieldLayoutEnabled && '7'"
-          class="field-container mb-3"
+          class="field-container flex-fill"
```

Comment:
form groups have a default margin, so you need to remove it. 
By removing mb-3 you actually just set it to mb-2